.travis.yml

```diff
@@ -71,6 +72,9 @@ before_install:
   # Remove Ruby directive from Gemfile as this image has 2.2.5
   - sed -i '/^ruby/d' Gemfile
   - mkdir -p ~/.apt-cache
+  - sudo add-apt-repository -y ppa:marutter/rdev
+  - sudo add-apt-repository -y ppa:staticfloat/juliareleases
+  - sudo add-apt-repository -y ppa:ondrej/golang
```

Comment:
It sounded like you moved these three from somewhere else. Shouldn't this include the removal from the other place?